src/app/form-builder/form-builder-groups/form-builder-groups.component.html

```diff
@@ -19,31 +19,39 @@
        [attr.data-id]="group.id">
 
     <div class="d-flex justify-content-between">
-      <div class="btn-group" role="group">
-        <button type="button"
-                class="btn btn-secondary delete"
-                (click)="onDelete(group.id)">
-          <i class="fa fa-trash"></i>
-        </button>
-        <button type="button"
-                class="btn btn-secondary shift-up"
-                (click)="onShiftGroupUp(group)">
-          <i class="fa fa-chevron-up"></i>
-        </button>
-        <button type="button"
-                class="btn btn-secondary shift-down"
-                (click)="onShiftGroupDown(group)">
-          <i class="fa fa-chevron-down"></i>
-        </button>
-        <button type="button"
-                class="btn btn-secondary select"
-                (click)="onSelectGroup(group)">
-          <i class="fa fa-list"
-             *ngIf="groupId !== group.id"></i>
-          <i class="fa fa-check-square-o"
-             *ngIf="groupId === group.id"></i>
-        </button>
-      </div>
+      <ul class="list-unstyled btn-group" role="group">
+        <li>
+          <button type="button"
+                  class="btn btn-secondary delete"
+                  (click)="onDelete(group.id)">
+            <i class="fa fa-trash"></i>
+          </button>
+        </li>
+        <li>
+          <button type="button"
+                  class="btn btn-secondary shift-up"
+                  (click)="onShiftGroupUp(group)">
+            <i class="fa fa-chevron-up"></i>
+          </button>
+        </li>
+        <li>
+          <button type="button"
+                  class="btn btn-secondary shift-down"
+                  (click)="onShiftGroupDown(group)">
+            <i class="fa fa-chevron-down"></i>
+          </button>
+        </li>
+        <li>
+          <button type="button"
+                  class="btn btn-secondary select"
+                  (click)="onSelectGroup(group)">
+            <i class="fa fa-list"
+               *ngIf="groupId !== group.id"></i>
+            <i class="fa fa-check-square-o"
+               *ngIf="groupId === group.id"></i>
+          </button>
+        </li>
+      </ul>
```

Comment:
i disagree with the change here. i'd like to keep the button group, maybe move to the button toolbar, as seen here https://getbootstrap.com/docs/4.0/components/button-group/ . according to bootstrap we can manage with accessibility okay!